pkg/k8sutils/templates/pod/spec.go

```diff
@@ -78,8 +78,29 @@ func getProbe(port int32, path string) corev1.ProbeHandler {
 	}
 }
 
+func MountConfigMaps(volumes []corev1.Volume, volumeMounts []corev1.VolumeMount,
+	configMaps []v1.ConfigMapReference) ([]corev1.Volume, []corev1.VolumeMount) {
+	for _, configMap := range configMaps {
+		volumes = append(volumes, corev1.Volume{
+			Name: configMap.Name,
```

Comment:
whether need to marshall or randomize this volume name, if the configmap name conflicts some other volume name (e.g. user specify a configmap named fe-meta)